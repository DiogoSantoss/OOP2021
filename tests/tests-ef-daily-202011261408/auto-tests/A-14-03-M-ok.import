SUPPLIER|P1|Publicações Europa-América|Lisboa, Portugal
SUPPLIER|P2|O’Reilly|Köln, Germany
CLIENT|R2|Jorge Figueiredo|Lisboa, Portugal
BOX|1C6H5OH|NORMAL|P1|2|20|100
BOX|4CO2|PERSONAL|P2|16|20|0
CONTAINER|7M5|EXPRESS|B4|P2|8|20|100
BOOK|B1937|How to fix almost everything|Satoshi Yamada|1928374650|P1|5|2|5
