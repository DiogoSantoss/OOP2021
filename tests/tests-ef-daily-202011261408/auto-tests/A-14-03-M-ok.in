5
1
4
Book1
tt
aa
ibbn1234
19
1
P1
3
CONT2
19
1
P1
NORMAL
DL
2
DCAIXA
19
1
P2
EXPRESS
1
0
0
