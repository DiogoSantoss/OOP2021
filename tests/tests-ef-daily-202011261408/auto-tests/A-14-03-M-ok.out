Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: BOX|1C6H5OH|P1|2|20|100|NORMAL
BOX|4CO2|P2|16|20|0|PERSONAL
CONTAINER|7M5|P2|8|20|100|EXPRESS|B4
BOOK|B1937|P1|5|2|5|How to fix almost everything|Satoshi Yamada|1928374650
Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Titulo do livro: Autor do livro: ISBN: Preço: Valor crítico para as existências: Identificador do fornecedor: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Nível de serviço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Valor crítico para as existências: Identificador do fornecedor: Tipo de serviço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: BOX|1C6H5OH|P1|2|20|100|NORMAL
BOX|4CO2|P2|16|20|0|PERSONAL
CONTAINER|7M5|P2|8|20|100|EXPRESS|B4
BOOK|B1937|P1|5|2|5|How to fix almost everything|Satoshi Yamada|1928374650
BOOK|Book1|P1|19|1|0|tt|aa|ibbn1234
CONTAINER|CONT2|P1|19|1|0|NORMAL|DL
BOX|DCAIXA|P2|19|1|0|EXPRESS
Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 